client/AdminUI/src/app/components/subscriptions/manage-subscription/manage-subscription.component.ts

```diff
@@ -762,6 +768,12 @@ export class ManageSubscriptionComponent implements OnInit, OnDestroy {
     });
   }
 
+  sendYearlyReport() {
+    this.subscriptionSvc.sendYearlyReport(this.subscription).subscribe(() => {
+      console.log('Sending yearly report.');
+    });
+  }
+
   /* Not in use yet
 
   viewYearlyReport() {
```

Comment:
Is this replaced with the added function above? Could remove commented out code.

Reply:
Ahh, didn't see that commented code. I'll remove those comments as that code is a bit different than what's in them.